docs/user-guide/how-to.md

```diff
@@ -148,3 +195,5 @@ Please refer to it for complete examples and guides.
 [terraform-docs GitHub Action]: https://github.com/terraform-docs/gh-actions
 [git hooks]: https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks
 [pre-commit-terraform]: https://github.com/antonbabenko/pre-commit-terraform
+[pre-commit]: https://pre-commit.com/
+[release tags]: https://github.com/terraform-docs/terraform-docs/releases
```

Comment:
Nit: I'd prefer to keep these footer links in lexicographic order, for easier to find in the future modifications.

Reply:
I considered sorting them, but the existing links don't look sorted to me, so I just added to the bottom.

Are you asking me to sort the entire block? I'm happy to, but to save yourself future reviewing effort I recommend we at least add a comment to say these should be sorted, or better find a linter that will auto-sort and add to the CI pipeline.

Reply:
Oh I thought they are already sorted, maybe I'm wrong, let me double check.

Anyways I prefer them to be in order, even if they are wrong at the moment. And to be honest I cannot think of a way to automate this and eventually it's not a very big deal if they are not sorted.

Reply:
Sorted now ✔️

Reply:
I apologize that I confused you, I should have made my intentions way more clearer! What I meant by sorted, was sort footer links by where they appear in the document. For example `[sections]` which is first defined in line 13 should be placed in footer before `[output]` which is first defined in line 75 and so on.

Reply:
> I apologize that I confused you, I should have made my intentions way more clearer!

You did say "in lexicographic order", which I took her to mean "alphabetically". I take it what you meant to say was "in order of first appearance".

That strikes me as an odd system though, given that, if I want to find the link for pre-commit, and it happens to have appeared earlier in the document, I might not know, and have to dig for it in the footer links. Alphabetical fixes that case, and is what I'd expect a linter/formatter tool to do, but I can change it back to how it was.

Reply:
That's a good point, then leave as is (alphabetically sorted) and we can see if that proves problematic in the future and we'll modify later. 👍

Reply:
Further comment 🙂 - adding links like this is quite a pain for a contributor, as I have to search through the entire page to find which if any link came before it, and then jump back to the bottom and insert my new link. And as I move paragraphs around and refactor my doc I have to adjust the ordering of the links.

It's not great UX for either the reader, contributor, or maintainer IMO, but I've added it correctly for consistency.